packages/utils/Logger.ts

```diff
@@ -227,15 +222,8 @@ class Logger {
 				const consoleObj = target.console ? target.console : console;
 				let items: any[] = [];
 
-				if (target.format) {
-					const format = level === LogLevel.Info && target.formatInfo ? target.formatInfo : target.format;
-
-					const s = sprintf(format, {
-						date_time: moment().format('YYYY-MM-DD HH:mm:ss'),
-						level: Logger.levelIdToString(level),
-						prefix: targetPrefix || '',
-						message: '',
-					});
```

Comment:
I can see why you moved this out of here, but to simplify the class interface and reduce boiler plate code when setting up the logger, I think it would be best to have it back here.

So the caller would simply do: `logger.format = (level) => '%(date_time)s: ${instancePrefix}%(prefix)s: %(message)s'`. And we'll take care of populating the variables on the Logger side